autoload/ChineseLinter.vim

```diff
@@ -70,12 +88,12 @@ endfunction
 
 function! s:add_to_qf(nr) abort
     let l:error_item = {
-                \ 'bufnr' : s:bufnr,
-                \ 'lnum' : s:linenr,
-                \ 'col' : s:colnr,
-                \ 'vcol' : 0,
-                \ 'text' : a:nr . ' ' . s:ERRORS[a:nr][0],
-                \ 'nr' : a:nr,
+                \ 'bufnr': s:bufnr                        ,
+                \ 'lnum' : s:linenr                       ,
+                \ 'col'  : s:colnr                        ,
+                \ 'vcol' : 0                              ,
+                \ 'text' : a:nr . ' ' . s:ERRORS[a:nr][0] ,
+                \ 'nr'   : a:nr                           ,
```

Comment:
no need to indent `,`

Reply:
The indents before `,` have been removed.